src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -454,6 +462,26 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
     this.alreadyHandlingDeviceChange = false;
   }
 
+  private async handleDeviceStreamEnded(kind: string, deviceId: string): Promise<void> {
+    if (
+      kind === 'video' &&
+      this.boundAudioVideoController &&
+      this.boundAudioVideoController.videoTileController.hasStartedLocalVideoTile()
+    ) {
+      this.boundAudioVideoController.videoTileController.stopLocalVideoTile();
+    }
+    await this.chooseInputDevice(kind, null, false);
```

Comment:
Isn't this statement a repetition of https://github.com/aws/amazon-chime-sdk-js/blob/master/src/devicecontroller/DefaultDeviceController.ts#L593

We have an 'ended' event which is called when the device is disconnected which sets the device to null